arith/gf2_128/src/gf2_ext128/avx.rs

```diff
@@ -318,3 +328,41 @@ fn mul_internal(a: &AVXGF2_128, b: &AVXGF2_128) -> AVXGF2_128 {
         v: unsafe { gfmul(a.v, b.v) },
     }
 }
+
+impl SimdField for AVXGF2_128 {
```

Comment:
We should probably rename it to something like AVXGF2Ext128.

Reply:
fair point, I was being lazy on this by directly taking advantage of `AVXGF2_128`, which can be viewed as a wrapper around `__m128i` - maybe we should just separate out a standalone thing like `AVXGF2x128` rather than the status quo...

Reply:
reverted by https://github.com/PolyhedraZK/Expander/pull/142/commits/4fd26abc44b17fa828b34d4bccf3a2eba86029d4